Common/Source/LKInterface/LKCustomKeyHandler.cpp

```diff
@@ -320,7 +320,10 @@ bool CustomKeyHandler(const int key) {
 		PlayResource(TEXT("IDR_WAV_CLICK"));
 		InputEvents::eventWaypointDetails(_T("select"));
 		return true;
-
+    case ckAirspaceLookup:
+        PlayResource(TEXT("IDR_WAV_CLICK"));
```

Comment:
why not the same sound ? cf. InputsEvents.cpp line 2019

Reply:
Just copied it from the case above, why make it different than other menue entries?

Reply:
ok, here it's not menu entrie, it's custom key... 
so if i configure custom with this, i don't have same sound than standart menu you have added ...

after that, agree, this is probably the right sound, for avoid this problem, you can change your change in InputEvents.cpp using same way used for "SONAR" or "OPTIMODE"